examples/js/MarchingCubes.js

```diff
@@ -735,6 +735,38 @@ THREE.MarchingCubes = function ( resolution, material, enableUvs, enableColors )
 
 	};
 
+	this.generateBufferGeometry = function() {
+
+		var start = 0, geo = new THREE.BufferGeometry();
+		var indices = [];
+		var posArray = new Float32Array();
+		var normArray = new Float32Array();
+		var colorArray = new Float32Array();
+		var uvArray = new Float32Array();
+		var scope = this;
+
+		var geo_callback = function( object ) {
+
+			if (scope.hasPositions) posArray = concatenate(Float32Array, posArray, object.positionArray, object.count*3);
```

Comment:
Where does the function `concatenate` comes from?

Reply:
from the part i forgot to copy in 😅 fixed